build/lib/fetch.ts

```diff
@@ -42,7 +42,7 @@ export function fetchUrls(urls: string[] | string, options: IFetchOptions): es.T
 }
 
 export async function fetchUrl(url: string, options: IFetchOptions, retries = 10, retryDelay = 1000): Promise<VinylFile> {
-	const verbose = !!options.verbose ?? (!!process.env['CI'] || !!process.env['BUILD_ARTIFACTSTAGINGDIRECTORY']);
+	const verbose = !!options.verbose;
```

Comment:
@bpasero I see you edited this line recently. The left hand side is a boolean value, which means the right hand side of the `??` can't be reached. My change keeps the existing behavior but please take a look to make sure there's not a better fix

Reply:
Thanks, I will push a change to be `verbose` in CI, which was the original intent.